cves/CVE-2013-2115.yml

```diff
@@ -45,7 +46,14 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  There are tags in Struts that accept an attribute called includeParams. It
+  only
+  accepts three values: none, get, and all. If a user creates a request using
+  Object-Graph Navigation Language in it, this will create a bad request to the
+  server, where a request may be able to manipulate context objects with
+  different priviledges than they should have.
```

Comment:
May want to fix the formatting here. What is the OGNL in context with struts?

Reply:
Do you think it might be worth mentioning here that this vulnerability was directly caused by the incomplete fix for CVE-2013-1966?

Reply:
Also, you might want to change "priviledges" to "privileges" :)